lib/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const MyApp(key: Key('value')));
```

Comment:
Dont need a key here, keys are optional and usually used for testing,

Reply:
G'day noob, thanks for the feedback, will resolve his right away